script/metrics.js

```diff
@@ -1,36 +1,60 @@
-
 /**
- * V6 weights
+ * Scoring curves
  */
-
-export const weights = {
+export const scoringGuides = {
+  v6: {
+    FCP: {weight: 0.15, median: 4000, falloff: 2000, auditId: 'first-contentful-paint', name: 'First Contentful Paint'},
+    SI: {weight: 0.15, median: 5800, falloff: 2900, auditId: 'speed-index', name: 'Speed Index'},
+    LCP: {weight: 0.25, median: 4000, falloff: 2000, auditId: 'largest-contentful-paint', name: 'Largest Contentful Paint'},
+    TTI: {weight: 0.15, median: 7300, falloff: 2900, auditId: 'interactive', name: 'Time to Interactive'},
+    TBT: {weight: 0.25, median: 600, falloff: 200, auditId: 'total-blocking-time', name: 'Total Blocking Time'},
+    CLS: {weight: 0.05, median: 0.25, falloff: 0.054, auditId: 'cumulative-layout-shift', name: 'Cumulative Layout Shift', units: 'unitless'},
+  },
   v5: {
-    FCP: 0.2,
-    SI: 0.26666,
-    FMP: 0.066666,
-    TTI: 0.33333,
-    FCI: 0.133333,
+    FCP: {weight: 0.2, median: 4000, falloff: 2000, auditId: 'first-contentful-paint', name: 'First Contentful Paint'},
+    SI: {weight: 0.26666, median: 5800, falloff: 2900, auditId: 'speed-index', name: 'Speed Index'},
+    FMP: {weight: 0.066666, median: 4000, falloff: 2000, auditId: 'first-meaningful-paint', name: 'First Meaningful Paint'},
+    TTI: {weight: 0.33333, median: 7300, falloff: 2900, auditId: 'interactive', name: 'Time to Interactive'},
+    FCI: {weight: 0.133333, median: 6500, falloff: 2900, auditId: 'first-cpu-idle', name: 'First CPU Idle'},
   },
+};
+
+// TODO need to support defining curves with p10 and podr.
+
+const scoringGuides_WIP_mobile = {
   v6: {
```

Comment:
I don't love how WET this file is now. :/  

we could compose objects like this pretty easily i think.

how about this file starts with something like

```js
const weights = {
  v5: {
    FCP: 0.2,
    SI: 0.26666,
    FMP: 0.066666,
    TTI: 0.33333,
    FCI: 0.133333,
  },
  v6: {
    FCP: 0.15,
    SI:  0.15,
    LCP: 0.25,
    TTI: 0.15,
    TBT: 0.25,
    CLS: 0.05
  },
};

const audits = {
  FCP: {auditId: 'first-contentful-paint', name: 'First Contentful Paint'},
  FMP: {auditId: 'first-meaningful-paint', name: 'First Meaningful Paint'},
  SI:  {auditId: 'speed-index', name: 'Speed Index'},
  TTI: {auditId: 'interactive', name: 'Time to Interactive'},
  FCI: {auditId: 'first-cpu-idle', name: 'First CPU Idle'},
  TBT: {auditId: 'total-blocking-time', name: 'Total Blocking Time'},
  LCP: {auditId: 'largest-contentful-paint', name: 'Largest Contentful Paint'},
  CLS: {auditId: 'cumulative-layout-shift', name: 'Cumulative Layout Shift', units: 'unitless'},
};


const curves = {
    FCP: {mobile: { median: 4000, falloff: 2000,  }, desktop: { median: 4000, falloff: 2000, } },
    FMP: {mobile: { median: 4000, falloff: 2000,  }, desktop: { median: 4000, falloff: 2000, } },
    SI: {mobile: { median: 5800, falloff: 2900,  },  desktop: { median: 5800, falloff: 2900, } },
    TTI: {mobile: { median: 7300, falloff: 2900,  }, desktop: { median: 7300, falloff: 2900, } },
    FCI: {mobile: { median: 6500, falloff: 2900,  }, desktop: { median: 6500, falloff: 2900, } },
    TBT: {mobile: { median: 600, falloff: 200,  },   desktop: { median: 600, falloff: 200, } },
    LCP: {mobile: { median: 4000, falloff: 2000,  }, desktop: { median: 4000, falloff: 2000, } },
    CLS: {mobile: { median: 0.25, falloff: 0.054, }, desktop: { median: 0.25, falloff: 0.054,} },
};
```

and then composes some scoringGuide object at the end.